src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +85,14 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables) {
+        String temp = new String();
+        for(Printable printable: printables) {
+            temp += printable.getPrintableString() + " ";
```

Comment:
For string concatenation in a loop, try to use StringJoiner for better readability and performance.